lib/gnat/services/service_responder.ex

```diff
@@ -0,0 +1,317 @@
+defmodule Gnat.Services.ServiceResponder do
+  @moduledoc false
+
+  require Logger
+  alias Gnat.Services.WireProtocol
+  @subscription_subject "$SRV.>"
```

Comment:
Should this be a dynamically calculation subject? it seems like the convention is to append the service name to the `$SRV` prefix? If our Responder subscribes to everything on this prefix, will it receive PING, STATS etc for other services?

Reply:
All services must respond to the action types without the service name. The `PING` and `INFO` commands are used to interrogate which services are available, so the responder needs to be listening on `$SRV.PING` and `$SRV.INFO` etc. There's a check below to see that if a service name is supplied, it matches the one defined in the responder.